to-sovereign/tests/to_sovereign_unit_tests.rs

```diff
@@ -758,3 +893,126 @@ fn register_token_nonfungible_token() {
                 .is_empty());
         })
 }
+
+#[test]
+fn execute_operation_no_header_verifier_address_registered() {
```

Comment:
Probably this test will be deleted later because `header_verifier_address` will be always valid if it comes in `init`